smart_open/__init__.py

```diff
@@ -33,17 +33,17 @@
 from smart_open import version  # noqa: E402
 from .smart_open_lib import open, parse_uri, smart_open, register_compressor  # noqa: E402
 
-deprecated_warning = """smart_open.s3_iter_bucket is deprecated and will stop functioning
+_WARNING = """smart_open.s3_iter_bucket is deprecated and will stop functioning
 in a future version. Please import iter_bucket from the smart_open.s3 module instead:
 
     from smart_open.s3 import iter_bucket as s3_iter_bucket
 
 """
-_warned = False
+_WARNED = False
 
 
 def s3_iter_bucket(
-    bucket_name,
+        bucket_name,
```

Comment:
Please do this for all the other parameters as well. Sorry, I thought it was obvious.

Reply:
Were you viewing an old diff maybe? They're all indented the same as far as I can see

Reply:
Please put the final `):` on a separate line. It helps readability.

Reply:
I'm happy to put the code in a style you prefer. Have y'all considered using something like Black to auto-format your code? It would certainly cut down on this kind of back and forth, and you could even configure a check to fail a PR so the style matches what you want before you even leave a review. 

I'd be happy to set that up in a separate PR if you're interested. For now I've matched the style that I found for other multi-line functions in the codebase. If you want it different, perhaps suggest an edit?

Reply:
Yes, we did consider black. Just didn't get to it yet :)

Reply:
Yeah, I still have a strong preference against automatic code formatting.

Reply:
@mpenkov It's interesting that you have a strong preference against it. I looked at some of your PRs and you're clearly a talented engineer. Why would you want to spend your valuable time nitpicking my code style in addition to the substance? When you auto-format code you don't always get the _exact_ style you want, but with OSS where consistency is hard to maintain, it eliminates an entire class of thing that you have to code-review for.

Reply:
I've addressed this [elsewhere](https://github.com/RaRe-Technologies/gensim/pull/2712#discussion_r364076359).

Under certain project or team dynamics, I can see how something like Black may be worth it. For example, if most PRs were from first-time contributors who are completely unfamiliar with the team conventions, or if every PR descended into a flame war about code style. Luckily, neither smart_open nor gensim seem to suffer from that problem.

Thanks for the compliment :)